apps/src/lib/levelbuilder/reference-guide-editor/NewReferenceGuideForm.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import RailsAuthenticityToken from '@cdo/apps/lib/util/RailsAuthenticityToken';
+import HelpTip from '@cdo/apps/lib/ui/HelpTip';
+
+const NewReferenceGuideForm = props => {
+  const {baseUrl} = props;
+  return (
+    <form action={baseUrl} method="post">
+      <RailsAuthenticityToken />
+      <h1>New Reference Guide</h1>
+      <label>
+        Key
+        <HelpTip>
+          <p>
+            The reference guide key is used in URLs and cannot be updated once
```

Comment:
In this help tip it is called a key and a slug. We also generally call it a slug or name on other pages. It seems like whatever language we use here it would be great to pick something that is clear to curriculum writers. I'm not sure which one that is

Reply:
Ah, oops, i copied it from the programming expressions page, which uses slug. I'll change it back to slug, it looks like other pages use that too.